syntax = "proto3";

package dpma;

message Patent {
  string id = 1;
  string applicant = 2;
  string application_date = 3;
  string title = 4;
  string inventor = 5;
  string class = 6;
  string publication_date = 7;
  string legal_agents = 8;
}
